src/query/functions/tests/it/scalars/testdata/function_list.txt

```diff
@@ -2440,6 +2440,7 @@ Functions overloads:
 1 map_cat(Map(Nothing) NULL, Map(Nothing) NULL) :: Map(Nothing) NULL
 2 map_cat(Map(T0, T1), Map(T0, T1)) :: Map(T0, T1)
 3 map_cat(Map(T0, T1) NULL, Map(T0, T1) NULL) :: Map(T0, T1) NULL
+0 map_contains_key(Map(T0, T1), T0) :: Boolean
```

Comment:
seems not correct
```
-0 map_contains_key(Map(T0, T1), T0) :: Boolean
+0 map_contains_key(Map(Nothing), T0) :: Boolean
+1 map_contains_key(Map(T0, T1), T0) :: Boolean
+2 map_contains_key(Map(T0, T1) NULL, T0 NULL) :: Boolean NULL
```

Reply:
May I ask, is it possible to deduce from the results of unit tests? Similar to when I was writing `map.txt`, I did so by examining the results of the unit tests to write the correct map.txt.

Reply:
You can write `map.txt` by extrapolating the results from the inputs of the unit tests, but since there are more result fields, we usually just copy the output from the run. Of course, you need to check whether the output is correct.